db/seeds.rb

```diff
@@ -38,28 +12,28 @@
 # ActiveRecord::Base.connection.reset_pk_sequence!('EventTemplate')
 
 template_list = [
-  ['blue','http://res.cloudinary.com/neddinn/image/upload/c_scale,h_92,w_101/v1442402582/blue_k0b4yt.png'],
-  ['teal','http://res.cloudinary.com/neddinn/image/upload/c_scale,h_92,w_101/v1442402583/teal_yajwze.png'],
-  ['red','http://res.cloudinary.com/neddinn/image/upload/c_scale,h_92,w_101/v1442402583/red_ug5h6q.png'],
-  ['orange','http://res.cloudinary.com/neddinn/image/upload/c_scale,h_92,w_101/v1442402583/orange_btitzu.png'],
-  ['lime','http://res.cloudinary.com/neddinn/image/upload/c_scale,h_92,w_101/v1442402583/lime_xlfi2l.png' ],
-  ['light-blue','http://res.cloudinary.com/neddinn/image/upload/c_scale,h_92,w_101/v1442402583/light-blue_o5ildf.png'],
-  ['grey', 'http://res.cloudinary.com/neddinn/image/upload/c_scale,h_92,w_101/v1442402582/grey_bfdrfw.png'],
-  ['pink', 'http://res.cloudinary.com/neddinn/image/upload/c_scale,h_92,w_101/v1442402583/pink_uzoho9.png'],
-  ['green','http://res.cloudinary.com/neddinn/image/upload/c_scale,h_92,w_101/v1442402582/green_r7ixce.png'],
-  ['cyan','http://res.cloudinary.com/neddinn/image/upload/c_scale,h_92,w_101/v1442402582/cyan_r5ptzy.png'],
-  ['brown','http://res.cloudinary.com/neddinn/image/upload/c_scale,h_92,w_101/v1442402582/brown_jzwgq6.png'],
-  ['purple','http://res.cloudinary.com/neddinn/image/upload/c_scale,h_92,w_101/v1442402583/purple_rryrhz.png']
+  ["blue","http://res.cloudinary.com/neddinn/image/upload/c_scale,h_92,w_101/v1442402582/blue_k0b4yt.png"],
+  ["teal","http://res.cloudinary.com/neddinn/image/upload/c_scale,h_92,w_101/v1442402583/teal_yajwze.png"],
+  ["red","http://res.cloudinary.com/neddinn/image/upload/c_scale,h_92,w_101/v1442402583/red_ug5h6q.png"],
+  ["orange","http://res.cloudinary.com/neddinn/image/upload/c_scale,h_92,w_101/v1442402583/orange_btitzu.png"],
+  ["lime","http://res.cloudinary.com/neddinn/image/upload/c_scale,h_92,w_101/v1442402583/lime_xlfi2l.png" ],
```

Comment:
Line is too long. [108/80]<br>Space missing after comma.<br>Space inside square brackets detected.